app/src/components/shared/Avatar.tsx

```diff
@@ -26,32 +28,63 @@ export const Avatar: React.FC<AvatarProps> = ({
   elevation = 0,
   borderRadius = 100,
   containerStyle,
-}) => (
-  <View
-    style={[
-      styles.container,
-      {
-        elevation,
-        height: size,
-        width: size,
-        borderRadius,
-      },
-      containerStyle,
-    ]}>
-    <Image
+  isLoading,
+}) => {
+  const avatarPlaceholder = require('../../../assets/versioned_initial_avatar.png').toString();
+
+  return isLoading ? (
+    <SkeletonPlaceholder>
+      <View
+        style={[
+          styles.container,
+          {
+            elevation,
+            height: size,
+            width: size,
+            borderRadius,
+          },
```

Comment:
Zróbmy jakieś zmienne na te style z użyciem useMemo, bo się powtarza to samo w 2 miejscach.